### PR TITLE
Adding option to load inline scripts

### LINKED ISSUE #3: Adding `eval` support

### PR DESCRIPTION
**Pull request by guillaumebriday**
fix https://github.com/stimulus-components/stimulus-content-loader/issues/3

### CODE REVIEW

**Comment by guillaumebriday** [comment]
Could you please review this PR @michaelbaudino? 🙏

File: src/index.js [hunk 1]
```diff
@@ -68,4 +73,13 @@ export default class extends Controller {
       clearInterval(this.refreshTimer)
     }
   }
+
+  loadScripts () {
+    this.element.querySelectorAll('script').forEach(content => {
+      const script = document.createElement('script')
+      script.text = content.innerHTML
```
Comment (michaelbaudino) [inline src/index.js]: What about this?

```suggestion
      script.innerHTML = content.innerHTML
```

File: src/index.js [hunk 2]
```diff
@@ -68,4 +73,13 @@ export default class extends Controller {
...
+
+      document.head.appendChild(script).parentNode.removeChild(script)
```
Comment (michaelbaudino) [inline src/index.js:82]: This is genius :star_struck:

Comment (guillaumebriday) [reply src/index.js:82]: Thanks `@rails/ujs`! 🙏

File: src/index.js (hunk 1 above)
Comment (guillaumebriday) [reply src/index.js]: You're right!

**Pull request merged**